_posts/2020-12-07-Unity-VR.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Building Virtual Spaces
+date: 2020-12-07 00:00:00 +0800
+author: Harvey
+---
+
+The discipline of computer science is typically likened to varying types of engineering such as mechanical, civil, electrical and so on. In terms of planning, this makes a lot of sense. 
+A software developers process is similar to an engineer’s “development loop” in the early stages in the sense that a developer (usually) plans out a broad architecture and <!--more-->
+then fine tunes the pieces within to make up the whole. The major difference between the two tends to revolve around execution and product delivery. While this is not always the case, an engineer typically has a physical object to show for their work whereas a software developer has a file. This notion is challenged with virtual reality.
+
+# Unity and the Oculus Quest 
+Recently I have been experimenting with the Oculus Quest 2 and Unity. After a few hiccups with incompatible assets, packages, and versions of unity, I was able to visit my virtual world for the first time. Interacting with my work with six degrees of freedom was a completely new experience for me. My first pass at the virtual space used the oculus integration asset and the XR Legacy Input Helpers.
+<div>
+    <img src="/assets/blog/2020-12-07/ball_play.gif">
```

Comment:
I wonder why the shadows look like this.. Toby explained it to me and I can't remember why, I think the scale of the objects..?